rollup.config.js

```diff
@@ -0,0 +1,63 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import {minify} from 'uglify-es';
+import replace from 'rollup-plugin-re';
+import resolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+
+function getRollupObject({file, minifying = false, format = 'umd'} = {}) {
+    const nonMinified = {
+        input: 'src/index.js',
+        output: {
+            format,
+            sourcemap: minifying,
+            name: 'XRegExp',
+            file: file || `dist/index-${format}${minifying ? '.min' : ''}.js`
+        },
+        plugins: [
+            replace({
+                patterns: [
+                    {
+                        match: /tools\/output/,
+                        test: 'module.exports = [',
+                        replace: 'export default ['
+                    }
+                ]
+            }),
+            babel({
+                babelrc: false,
+                presets: [
+                    ['env', {
+                        modules: false,
+                        exclude: ['transform-es2015-literals']
+                    }]
+                ],
+                plugins: [
+                    'array-includes',
+                    'transform-xregexp'
+                ]
+            }),
```

Comment:
Is there a way to reduce the duplication between these babel options and the `.babelrc` file? Perhaps rollup could generate the files in `lib/` instead of the `npm run babel` script that's part of `npm run prebuild`?

Reply:
Re: one aspect of `.babelrc` duplication, I could probably just import the `.babelrc` file here and adapt it...

However, as far as Rollup replacing the script acting on `lib`, no, I don't think so. Rollup is concerned with rolling up the files into a single file whereas Babel just provides the backward-compatible code (and which Rollup itself uses as a dependency).

Reply:
That sounds good. It looks like we should just move the contents of `.babelrc` into the `babel` property of `package.json`. That way, we can import/require it easily from `rollup.config.js`.

https://babeljs.io/docs/usage/babelrc

Reply:
Yes, that should work, too...